examples/riscv/Cargo.toml

```diff
@@ -6,10 +6,25 @@ edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[features]
+tcp = ["smoltcp"]
+default = []
```

Comment:
As an example crate, I'd like to have smoltcp enabled by default, so that users can see its usage with no more effort.

Reply:
Enabling the tcp server will block CI.